packages/dds/shared-object-base/src/test/attachingBindingAndConnecting.spec.ts

```diff
@@ -450,16 +455,17 @@ describe("SharedObject attaching binding and connecting", () => {
 					connected && sharedObject.isAttached(),
 					"connected",
 				);
-			}),
-		);
+			});
+		}
 
 		it("isAttached with detached transition to attach runtime", async () => {
 			const runtimeEvents = new TypedEventEmitter<IFluidDataStoreRuntimeEvents>();
+
 			let didAttach = 0;
 			let attachCalled = false;
 			const { overrides, sharedObject } = createTestSharedObject({
 				runtime: {
-					...(runtimeEvents as any),
+					...(runtimeEvents as unknown as Overridable<IFluidDataStoreRuntime>),
```

Comment:
Using the same pattern found already in other places in this file. Trying to use `...runtimeEvents` on its own results in mismatched typing issues along the lines of the one below, which I don't quite understand or know how to fix:

```
<several more entries...>
Type 'TypedEventTransform<TypedEventEmitter<IFluidDataStoreRuntimeEvents>, IFluidDataStoreRuntimeEvents>' is not assignable to type 'TransformedEvent<IFluidDataStoreRuntime, "disconnected", []>'.
  Types of parameters 'event' and 'event' are incompatible.
    Type '"disconnected"' is not assignable to type '"newListener" | "removeListener"'.ts(2322)
```